integration/testrunner/grpc.go

```diff
@@ -170,13 +179,21 @@ func testGRPCEndpoint(ctx context.Context, t *testing.T, g *GRPC) {
 			}
 			t.Logf("Received value %v", place.Value)
 		}
-		query := g.makeQueryForGRPCStream(id)
-		queryIfAuditLogsExistWithRetries(ctx, t, query, g.Config, "server_stream_fibonacci", int64(places))
+		query := g.makeQueryForGRPC(id)
+		t.Logf("querying BigQuery:\n%s", query.Q)
+		wantNum := places
+		logEntries := queryAuditLogsWithRetries(ctx, t, query, g.Config, "server_stream_fibonacci", wantNum)
+		checkNum(t, logEntries, wantNum)
+		for i := 0; i < wantNum; i++ {
+			t.Logf("diffing LogEntry for index %v", i)
```

Comment:
Is this log useful? What's the meaning of the index?

Reply:
I was refactoring the original logic written by Qinhang. With index, we can clearly know how many log entries in a stream requests are different from expected. As far as I can see, I'm also fine to remove this log.